lambda_uploader/package.py

```diff
@@ -66,6 +67,9 @@ def clean_zipfile(self):
         if os.path.isfile(self.zip_file):
             os.remove(self.zip_file)
 
+    def check_current_package_size(self):
```

Comment:
Since this is being used in an if statement I think it should be named to reflect the result. Something similar to package_over_max() is a bit more descriptive.